sources/platform/actors/publishing/monetize.mdx

```diff
@@ -13,13 +13,131 @@ Apify Store allows you to monetize your web scraping and automation projects by
+### Rental pricing model
+
+With the rental model, you can specify a free trial period and a monthly rental price. After the trial, users with an [Apify paid plan](https://apify.com/pricing) can continue using your Actor by paying the monthly fee. You can receive 80% of the total rental fees collected each month.
+
+<details>
+    <summary>Example - rental pricing model</summary>
+
+You make your Actor rental with 7-day free trial and then $30/month. During the first calendar month, three users start to use your Actor:
+
+1. First user, on the Apify paid plan, starts the free trial on 15th
+1. Second user, on the Apify paid plan, starts the free trial on 25th
+1. Third user, on the Apify free plan, start the free trial on 20th
+
+The first user pays their first rent 7 days after the free trial, i.e., on the 22nd of the month. The second user only starts paying the rent next month. The third user is on the Apify free plan, so after the free trial ends on the 27th of the month, they are not charged and cannot use the Actor further until they get a paid plan. Your profit is computed only from the first user. They were charged $30, so 80% of this goes to you, i.e., _0.8 * 30 = $24_.
+</details>
+
+### Pay-per-result pricing model
+
+In this model, you set a price per 1,000 results. Users are charged based on the number of results your Actor produces. Your profit is calculated as 80% of the revenue minus platform usage costs. The formula is:
+
+`(0.8 * revenue) - costs = profit`
+
+#### Pay-per-result unit pricing for cost computation
+
+| Service                         | Unit price                 |
+|:--------------------------------|:---------------------------|
+| Compute unit                    | **$0.4** / CU              |
+| Residential proxies             | **$13** / GB               |
+| SERPs proxy                     | **$3** / 1,000 SERPs       |
+| Data transfer - external        | **$0.20** / GB             |
+| Data transfer - internal        | **$0.05** / GB             |
+| Dataset - timed storage         | **$1.00** / 1,000 GB-hours |
+| Dataset - reads                 | **$0.0004** / 1,000 reads  |
+| Dataset - writes                | **$0.005** / 1,000 writes  |
+| Key-value store - timed storage | **$1.00** / 1,000 GB-hours |
+| Key-value store - reads         | **$0.005** / 1,000 reads   |
+| Key-value store - writes        | **$0.05** / 1,000 writes   |
+| Key-value store - lists         | **$0.05** / 1,000 lists    |
+| Request queue - timed storage   | **$4.00** / 1,000 GB-hours |
+| Request queue - reads           | **$0.004** / 1,000 reads   |
+| Request queue - writes          | **$0.02** / 1,000 writes   |
+
+Only revenue and cost for Apify customers on paid plans are taken into consideration when computing your profit. Users on free plans are not reflected there, although you can see statistics about the potential revenue of users that are currently on free plans in Actor Insights in the Apify Console.
```

Comment:
`although you can see statistics about the potential revenue of users that are currently on free plans in Actor Insights in the Apify Console` This is no longer true, let's remove this part please.